src/css/styles.css

```diff
@@ -62,17 +66,33 @@ canvas {
 }
 
 .grid {
-  margin-left: 45px;
+  column-count: 3;
+  column-gap: 1em;
+  column-fill: balance;
+
+}
+
+.grid:after {
+  content: '';
+  display: block;
+  clear: both;
+}
+
+.grid-item {
+  float: left;
+  width: 100%;
+  break-inside: avoid;
```

Comment:
Curious what this does. I commented it out and it makes the bottom of each section appear above the tile. Interesting!

Reply:
I am not totally following what's going on here either. I think we should still try and figure out how to relink the packery package once we get more familiar with how webpack works.